gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -146,8 +146,9 @@ typedef struct _GstTensorFilterProperties
   int invoke_dynamic; /**< True for supporting invoke with flexible output. */
   uint32_t suspend; /**< Timeout (ms) for suspend. (Unload the framework) */
 
-  NNSFilterInvokeAsyncCallback invoke_async_callback; /**< If a filter is configured to emit output frames asynchronously (e.g., N output frames for 1 input frame), tensor_filter framework implementation sets and uses this. */
-  void *async_handle; /**< Internal data structure for invoke_async_callback, configured by tensor_filter framework */
+  int invoke_async; /**< The sub-plugin must support asynchronous output to use this option. If set to TRUE, the sub-plugin can generate multiple outputs asynchronously per single input. Otherwise, only synchronous single-output is expected and async callback/handle are ignored. */
```

Comment:
(repost) This breaks ABI. Move new elements to the end.

Reply:
Thank you. I've updated.